src/Api/WebClient.php

```diff
@@ -149,8 +152,12 @@ public function getShipments(): ?array
         return $this->request('GET', $url);
     }
 
-    public function createShipment(ShipmentInterface $shipment): array
-    {
+    public function createShipment(
+        ShipmentInterface $shipment,
+        ?ShippingExportInterface $shippingExport = null,
+    ): array {
+        $this->shippingExport = $shippingExport;
```

Comment:
There is no need for putting this as a class property. You can put this into the dependent methods as a parameter.

Reply:
Done 👍